### PR TITLE
Firefox_nss: Revert unnecessary x11utils::turn_off_gnome_screensaver call

### PR DESCRIPTION
**Pull request by jouyingbin**
Firefox_nss calls x11utils::turn_off_gnome_screensaver is a simple
script_run 'gsettings set org.gnome.desktop.session idle-delay 0';

1. In order to avoid no terminal is shown the string which is just typed "into the void"
   of the desktop session. Therefore there is also never an output on the serial port.
2. This patch can also fixed the side effect error about case running after firefox_nss.

Error run: https://openqa.suse.de/tests/3022911#step/firefox_nss/4

- Related ticket: 
https://progress.opensuse.org/issues/53978 
https://progress.opensuse.org/issues/53651
- Needles: NA
- Verification run: 
(firefox_nss -> **seahorse_sshkey**) http://10.163.2.52/tests/297
(firefox_nss -> **hexchat_ssl**) http://10.163.2.52/tests/298

### CODE REVIEW

File: tests/fips/mozilla_nss/firefox_nss.pm [hunk 1]
```diff
@@ -28,16 +28,18 @@ sub quit_firefox {
...
+    my ($self) = @_;
 
     # Define FIPS password for firefox, and it should be consisted by:
     # - at least 8 characters
     # - at least one upper case
     # - at least one non-alphabet-non-number character (like: @-.=%)
     my $fips_password = 'openqa@SUSE';
 
-    # Turn off screensaver before launch firefox in order to avoid the screensaver block
-    turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome');
```
Comment (lilyeyes) [inline tests/fips/mozilla_nss/firefox_nss.pm]: yes, this error https://openqa.suse.de/tests/3022911#step/firefox_nss/4 *is* introduced by `turn_off_gnome_screensaver`
, and you explanation makes sense. 
So, this invalid line can be deleted.

But I can not understand why the case `seahorse_sshkey` can pass after deleting the above line and
maybe adding line `select_console 'root-console';` make the "seahorse_sshkey" pass.

Whatever, all the changes are no harm and work.

Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm]: 1. We added this function call to be a workaround before to avoid the screensaver lock while launching the firefox, but actually turn_off_gnome_screensaver is not working at all in the case, thus our solution was not a real fixing here.

2. After adding this function call, there is no console which can accept the code to turn off gnome screensaver, thus it just typed "into the void" of the desktop session, but openQA is still waiting for a wait_serial string matching, it causes error I observed in the log. But afterward it does not lead to the case Fail, the firefox_nss case is still Passed in this situation.

3. On the other hand, we can use select_console 'root-console' or xterm to type the command script_run 'gsettings set org.gnome.desktop.session idle-delay 0'; 
   e.g., add x11_start_program('xterm') and then run turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome'), I think it can also fixed the problem.

4. After testing, it is still Fail after adding line select_console 'root-console' in "seahorse_sshkey" too. Maybe it should be a backend problem.

To sum up, in fact, we don't need to really turn off the screen saver at all, I think the real fixing is to revert the turn_off_gnome_screensaver function call.

Comment (lilyeyes) [reply tests/fips/mozilla_nss/firefox_nss.pm]: for `4.` your verify run is PASS not "Fail".

Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm]: Yes, the verification run is PASSED now. :) 
I mean if I only add **select_console 'root-console'** in `seahorse_sshkey`, but did not revert the **turn_off_gnome_screensaver** call in `firefox_nss` , the case `seahorse_sshkey` is still Fail.

File: tests/fips/mozilla_nss/firefox_nss.pm [hunk 2]
```diff
@@ -28,16 +28,18 @@ sub quit_firefox {
...
+    select_console 'root-console';
+    zypper_call 'in firefox';
```
Comment (wnereiz) [inline tests/fips/mozilla_nss/firefox_nss.pm]: Is it necessary to install firefox? It is there by default for gnome desktop afaik.

File: tests/fips/mozilla_nss/firefox_nss.pm (hunk 1 above)
Comment (wnereiz) [reply tests/fips/mozilla_nss/firefox_nss.pm]: It does not explain why **turn_off_gnome_screensaver** make following `select_console` failed. If we want to dive deeper we should check the code which implement **turn_off_gnome_screensaver** in x11utils. But as a work around it is acceptable to simply remove it from the code.

**Comment by wnereiz** [comment]
The change does not explain why **turn_off_gnome_screensaver** make following `select_console` failed. If we want to dive deeper we should check the code which implement **turn_off_gnome_screensaver** in x11utils. But as a work around it is acceptable to simply remove it from the code.

File: tests/fips/mozilla_nss/firefox_nss.pm (hunk 2 above)
Comment (jouyingbin) [reply tests/fips/mozilla_nss/firefox_nss.pm]: Yes, firefox is installed by default. Thanks for reminding and Updated.

**Comment by jouyingbin** [comment]
> The change does not explain why **turn_off_gnome_screensaver** make following `select_console` failed. If we want to dive deeper we should check the code which implement **turn_off_gnome_screensaver** in x11utils. But as a work around it is acceptable to simply remove it from the code.

Firefox_nss calls x11utils::turn_off_gnome_screensaver is a simple script_run 'gsettings set org.gnome.desktop.session idle-delay 0';

We can either add select_console 'root-console' or xterm to type the command script_run 'gsettings set org.gnome.desktop.session idle-delay 0';  e.g., add x11_start_program('xterm') and then run turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome'), I think it can also fix the problem too.

Currently, I don't see the reason that we need to really turn off the screen saver, so I plan to revert the turn_off_gnome_screensaver function call. 

Thanks.

**Comment by lemon-suse** [comment]
LGTM.
